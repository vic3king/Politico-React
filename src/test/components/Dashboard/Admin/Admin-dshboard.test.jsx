import React from 'react';
import { shallow } from 'enzyme';
// import { Provider } from 'react-redux';
// import thunk from 'redux-thunk';
// import configureMockStore from 'redux-mock-store';
import AdminPage from '../../../../components/Dashboard/Admin/Admin';

// const middlewares = [thunk];
// const mockStore = configureMockStore(middlewares);
// const store = mockStore({});

const props = {
  offices: {
    officeList: [{ id: 'id', office: 'office' }],
  },
  parties: {
    partiesList: [{ id: 'id', party: 'office' }],
  },
  isLoadingReducer: { loader: true },
};

const AdminPageComponent = <AdminPage {...props} />;

describe('<AdminPage />', () => {
  it('renders admin page', () => {
    localStorage.setItem('user', '{ "id": 2 }');
    const wrapper = shallow(AdminPageComponent);
    expect(wrapper.find('div'));
  });
});

describe('<AdminPage />', () => {
  it('renders Admin page', () => {
    const wrapper = shallow(AdminPageComponent);
    expect(wrapper.find('div'));
    wrapper.instance().showOfficeModal();
    wrapper.instance().hideOfficeModal();
    wrapper.instance().showPartyModal();
    wrapper.instance().hidePartyModal();
    wrapper.instance().handleChangeView();
    wrapper.instance().updateDeletePartyState(0);
    wrapper.instance().updatePartiesState([]);
  });
});
