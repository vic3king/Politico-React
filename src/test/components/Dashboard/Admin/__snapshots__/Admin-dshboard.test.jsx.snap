// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminPage should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AdminPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <Redirect
          to="/login"
        />,
        false,
        false,
        false,
        <NavBar
          show={true}
        >
          <Button
            className="reset_button"
            id="Dashboard"
            onClick={[Function]}
            value="Dashboard"
          />
          <LiTag
            to="/logout"
            value="Logout"
          />
        </NavBar>,
        <div
          className="main"
        >
          <SideBar
            ButtonFour={
              <Button
                className="profile-btn"
                id="requests"
                onClick={[Function]}
                value="Reveiw Requests"
              />
            }
            ButtonOne={
              <Button
                className="profile-btn"
                id="offic"
                onClick={[Function]}
                value="Add Office"
              />
            }
            ButtonThree={
              <Button
                className="profile-btn"
                id="results"
                onClick={[Function]}
                value="Results"
              />
            }
            ButtonTwo={
              <Button
                className="profile-btn"
                id="party"
                onClick={[Function]}
                value="Add Party"
              />
            }
            user={Object {}}
          />
          <div
            className="boxed"
          >
            <section
              className="main-section"
            >
              <div>
                <h4
                  className="page-title"
                >
                  Open Positions
                  <hr />
                </h4>
              </div>
              <ProfileTopSectionCard
                offices={
                  Array [
                    Object {
                      "name": "",
                    },
                    Object {
                      "name": "",
                    },
                  ]
                }
                value="View Running Candidates"
              />
            </section>
            <section
              className="maim-section"
            >
              <div>
                <h4
                  className="page-title"
                >
                  Political Parties
                  <hr />
                </h4>
              </div>
              <ProfileBottomSectionCard
                hideOfficeModal={[Function]}
                hidePartyModal={[Function]}
                parties={
                  Array [
                    Object {
                      "id": 1,
                      "name": "",
                    },
                    Object {
                      "id": 2,
                      "name": "",
                    },
                  ]
                }
                updateDelete={[Function]}
                updatePartiesName={[Function]}
              />
            </section>
          </div>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "to": "/login",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      false,
      false,
      false,
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Button
              className="reset_button"
              id="Dashboard"
              onClick={[Function]}
              value="Dashboard"
            />,
            <LiTag
              to="/logout"
              value="Logout"
            />,
          ],
          "show": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "className": "reset_button",
              "id": "Dashboard",
              "onClick": [Function],
              "value": "Dashboard",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "to": "/logout",
              "value": "Logout",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <SideBar
              ButtonFour={
                <Button
                  className="profile-btn"
                  id="requests"
                  onClick={[Function]}
                  value="Reveiw Requests"
                />
              }
              ButtonOne={
                <Button
                  className="profile-btn"
                  id="offic"
                  onClick={[Function]}
                  value="Add Office"
                />
              }
              ButtonThree={
                <Button
                  className="profile-btn"
                  id="results"
                  onClick={[Function]}
                  value="Results"
                />
              }
              ButtonTwo={
                <Button
                  className="profile-btn"
                  id="party"
                  onClick={[Function]}
                  value="Add Party"
                />
              }
              user={Object {}}
            />,
            <div
              className="boxed"
            >
              <section
                className="main-section"
              >
                <div>
                  <h4
                    className="page-title"
                  >
                    Open Positions
                    <hr />
                  </h4>
                </div>
                <ProfileTopSectionCard
                  offices={
                    Array [
                      Object {
                        "name": "",
                      },
                      Object {
                        "name": "",
                      },
                    ]
                  }
                  value="View Running Candidates"
                />
              </section>
              <section
                className="maim-section"
              >
                <div>
                  <h4
                    className="page-title"
                  >
                    Political Parties
                    <hr />
                  </h4>
                </div>
                <ProfileBottomSectionCard
                  hideOfficeModal={[Function]}
                  hidePartyModal={[Function]}
                  parties={
                    Array [
                      Object {
                        "id": 1,
                        "name": "",
                      },
                      Object {
                        "id": 2,
                        "name": "",
                      },
                    ]
                  }
                  updateDelete={[Function]}
                  updatePartiesName={[Function]}
                />
              </section>
            </div>,
            null,
            null,
          ],
          "className": "main",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "ButtonFour": <Button
                className="profile-btn"
                id="requests"
                onClick={[Function]}
                value="Reveiw Requests"
              />,
              "ButtonOne": <Button
                className="profile-btn"
                id="offic"
                onClick={[Function]}
                value="Add Office"
              />,
              "ButtonThree": <Button
                className="profile-btn"
                id="results"
                onClick={[Function]}
                value="Results"
              />,
              "ButtonTwo": <Button
                className="profile-btn"
                id="party"
                onClick={[Function]}
                value="Add Party"
              />,
              "user": Object {},
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <section
                  className="main-section"
                >
                  <div>
                    <h4
                      className="page-title"
                    >
                      Open Positions
                      <hr />
                    </h4>
                  </div>
                  <ProfileTopSectionCard
                    offices={
                      Array [
                        Object {
                          "name": "",
                        },
                        Object {
                          "name": "",
                        },
                      ]
                    }
                    value="View Running Candidates"
                  />
                </section>,
                <section
                  className="maim-section"
                >
                  <div>
                    <h4
                      className="page-title"
                    >
                      Political Parties
                      <hr />
                    </h4>
                  </div>
                  <ProfileBottomSectionCard
                    hideOfficeModal={[Function]}
                    hidePartyModal={[Function]}
                    parties={
                      Array [
                        Object {
                          "id": 1,
                          "name": "",
                        },
                        Object {
                          "id": 2,
                          "name": "",
                        },
                      ]
                    }
                    updateDelete={[Function]}
                    updatePartiesName={[Function]}
                  />
                </section>,
              ],
              "className": "boxed",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div>
                      <h4
                        className="page-title"
                      >
                        Open Positions
                        <hr />
                      </h4>
                    </div>,
                    <ProfileTopSectionCard
                      offices={
                        Array [
                          Object {
                            "name": "",
                          },
                          Object {
                            "name": "",
                          },
                        ]
                      }
                      value="View Running Candidates"
                    />,
                  ],
                  "className": "main-section",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <h4
                        className="page-title"
                      >
                        Open Positions
                        <hr />
                      </h4>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Open Positions",
                          <hr />,
                        ],
                        "className": "page-title",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Open Positions",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "hr",
                        },
                      ],
                      "type": "h4",
                    },
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "offices": Array [
                        Object {
                          "name": "",
                        },
                        Object {
                          "name": "",
                        },
                      ],
                      "value": "View Running Candidates",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "section",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div>
                      <h4
                        className="page-title"
                      >
                        Political Parties
                        <hr />
                      </h4>
                    </div>,
                    <ProfileBottomSectionCard
                      hideOfficeModal={[Function]}
                      hidePartyModal={[Function]}
                      parties={
                        Array [
                          Object {
                            "id": 1,
                            "name": "",
                          },
                          Object {
                            "id": 2,
                            "name": "",
                          },
                        ]
                      }
                      updateDelete={[Function]}
                      updatePartiesName={[Function]}
                    />,
                  ],
                  "className": "maim-section",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <h4
                        className="page-title"
                      >
                        Political Parties
                        <hr />
                      </h4>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Political Parties",
                          <hr />,
                        ],
                        "className": "page-title",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Political Parties",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "hr",
                        },
                      ],
                      "type": "h4",
                    },
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "hideOfficeModal": [Function],
                      "hidePartyModal": [Function],
                      "parties": Array [
                        Object {
                          "id": 1,
                          "name": "",
                        },
                        Object {
                          "id": 2,
                          "name": "",
                        },
                      ],
                      "updateDelete": [Function],
                      "updatePartiesName": [Function],
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "section",
              },
            ],
            "type": "div",
          },
          null,
          null,
        ],
        "type": "div",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <Redirect
            to="/login"
          />,
          false,
          false,
          false,
          <NavBar
            show={true}
          >
            <Button
              className="reset_button"
              id="Dashboard"
              onClick={[Function]}
              value="Dashboard"
            />
            <LiTag
              to="/logout"
              value="Logout"
            />
          </NavBar>,
          <div
            className="main"
          >
            <SideBar
              ButtonFour={
                <Button
                  className="profile-btn"
                  id="requests"
                  onClick={[Function]}
                  value="Reveiw Requests"
                />
              }
              ButtonOne={
                <Button
                  className="profile-btn"
                  id="offic"
                  onClick={[Function]}
                  value="Add Office"
                />
              }
              ButtonThree={
                <Button
                  className="profile-btn"
                  id="results"
                  onClick={[Function]}
                  value="Results"
                />
              }
              ButtonTwo={
                <Button
                  className="profile-btn"
                  id="party"
                  onClick={[Function]}
                  value="Add Party"
                />
              }
              user={Object {}}
            />
            <div
              className="boxed"
            >
              <section
                className="main-section"
              >
                <div>
                  <h4
                    className="page-title"
                  >
                    Open Positions
                    <hr />
                  </h4>
                </div>
                <ProfileTopSectionCard
                  offices={
                    Array [
                      Object {
                        "name": "",
                      },
                      Object {
                        "name": "",
                      },
                    ]
                  }
                  value="View Running Candidates"
                />
              </section>
              <section
                className="maim-section"
              >
                <div>
                  <h4
                    className="page-title"
                  >
                    Political Parties
                    <hr />
                  </h4>
                </div>
                <ProfileBottomSectionCard
                  hideOfficeModal={[Function]}
                  hidePartyModal={[Function]}
                  parties={
                    Array [
                      Object {
                        "id": 1,
                        "name": "",
                      },
                      Object {
                        "id": 2,
                        "name": "",
                      },
                    ]
                  }
                  updateDelete={[Function]}
                  updatePartiesName={[Function]}
                />
              </section>
            </div>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "to": "/login",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        false,
        false,
        false,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Button
                className="reset_button"
                id="Dashboard"
                onClick={[Function]}
                value="Dashboard"
              />,
              <LiTag
                to="/logout"
                value="Logout"
              />,
            ],
            "show": true,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "className": "reset_button",
                "id": "Dashboard",
                "onClick": [Function],
                "value": "Dashboard",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "to": "/logout",
                "value": "Logout",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <SideBar
                ButtonFour={
                  <Button
                    className="profile-btn"
                    id="requests"
                    onClick={[Function]}
                    value="Reveiw Requests"
                  />
                }
                ButtonOne={
                  <Button
                    className="profile-btn"
                    id="offic"
                    onClick={[Function]}
                    value="Add Office"
                  />
                }
                ButtonThree={
                  <Button
                    className="profile-btn"
                    id="results"
                    onClick={[Function]}
                    value="Results"
                  />
                }
                ButtonTwo={
                  <Button
                    className="profile-btn"
                    id="party"
                    onClick={[Function]}
                    value="Add Party"
                  />
                }
                user={Object {}}
              />,
              <div
                className="boxed"
              >
                <section
                  className="main-section"
                >
                  <div>
                    <h4
                      className="page-title"
                    >
                      Open Positions
                      <hr />
                    </h4>
                  </div>
                  <ProfileTopSectionCard
                    offices={
                      Array [
                        Object {
                          "name": "",
                        },
                        Object {
                          "name": "",
                        },
                      ]
                    }
                    value="View Running Candidates"
                  />
                </section>
                <section
                  className="maim-section"
                >
                  <div>
                    <h4
                      className="page-title"
                    >
                      Political Parties
                      <hr />
                    </h4>
                  </div>
                  <ProfileBottomSectionCard
                    hideOfficeModal={[Function]}
                    hidePartyModal={[Function]}
                    parties={
                      Array [
                        Object {
                          "id": 1,
                          "name": "",
                        },
                        Object {
                          "id": 2,
                          "name": "",
                        },
                      ]
                    }
                    updateDelete={[Function]}
                    updatePartiesName={[Function]}
                  />
                </section>
              </div>,
              null,
              null,
            ],
            "className": "main",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "ButtonFour": <Button
                  className="profile-btn"
                  id="requests"
                  onClick={[Function]}
                  value="Reveiw Requests"
                />,
                "ButtonOne": <Button
                  className="profile-btn"
                  id="offic"
                  onClick={[Function]}
                  value="Add Office"
                />,
                "ButtonThree": <Button
                  className="profile-btn"
                  id="results"
                  onClick={[Function]}
                  value="Results"
                />,
                "ButtonTwo": <Button
                  className="profile-btn"
                  id="party"
                  onClick={[Function]}
                  value="Add Party"
                />,
                "user": Object {},
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <section
                    className="main-section"
                  >
                    <div>
                      <h4
                        className="page-title"
                      >
                        Open Positions
                        <hr />
                      </h4>
                    </div>
                    <ProfileTopSectionCard
                      offices={
                        Array [
                          Object {
                            "name": "",
                          },
                          Object {
                            "name": "",
                          },
                        ]
                      }
                      value="View Running Candidates"
                    />
                  </section>,
                  <section
                    className="maim-section"
                  >
                    <div>
                      <h4
                        className="page-title"
                      >
                        Political Parties
                        <hr />
                      </h4>
                    </div>
                    <ProfileBottomSectionCard
                      hideOfficeModal={[Function]}
                      hidePartyModal={[Function]}
                      parties={
                        Array [
                          Object {
                            "id": 1,
                            "name": "",
                          },
                          Object {
                            "id": 2,
                            "name": "",
                          },
                        ]
                      }
                      updateDelete={[Function]}
                      updatePartiesName={[Function]}
                    />
                  </section>,
                ],
                "className": "boxed",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <div>
                        <h4
                          className="page-title"
                        >
                          Open Positions
                          <hr />
                        </h4>
                      </div>,
                      <ProfileTopSectionCard
                        offices={
                          Array [
                            Object {
                              "name": "",
                            },
                            Object {
                              "name": "",
                            },
                          ]
                        }
                        value="View Running Candidates"
                      />,
                    ],
                    "className": "main-section",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <h4
                          className="page-title"
                        >
                          Open Positions
                          <hr />
                        </h4>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            "Open Positions",
                            <hr />,
                          ],
                          "className": "page-title",
                        },
                        "ref": null,
                        "rendered": Array [
                          "Open Positions",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {},
                            "ref": null,
                            "rendered": null,
                            "type": "hr",
                          },
                        ],
                        "type": "h4",
                      },
                      "type": "div",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "offices": Array [
                          Object {
                            "name": "",
                          },
                          Object {
                            "name": "",
                          },
                        ],
                        "value": "View Running Candidates",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": "section",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <div>
                        <h4
                          className="page-title"
                        >
                          Political Parties
                          <hr />
                        </h4>
                      </div>,
                      <ProfileBottomSectionCard
                        hideOfficeModal={[Function]}
                        hidePartyModal={[Function]}
                        parties={
                          Array [
                            Object {
                              "id": 1,
                              "name": "",
                            },
                            Object {
                              "id": 2,
                              "name": "",
                            },
                          ]
                        }
                        updateDelete={[Function]}
                        updatePartiesName={[Function]}
                      />,
                    ],
                    "className": "maim-section",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <h4
                          className="page-title"
                        >
                          Political Parties
                          <hr />
                        </h4>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            "Political Parties",
                            <hr />,
                          ],
                          "className": "page-title",
                        },
                        "ref": null,
                        "rendered": Array [
                          "Political Parties",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {},
                            "ref": null,
                            "rendered": null,
                            "type": "hr",
                          },
                        ],
                        "type": "h4",
                      },
                      "type": "div",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "hideOfficeModal": [Function],
                        "hidePartyModal": [Function],
                        "parties": Array [
                          Object {
                            "id": 1,
                            "name": "",
                          },
                          Object {
                            "id": 2,
                            "name": "",
                          },
                        ],
                        "updateDelete": [Function],
                        "updatePartiesName": [Function],
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": "section",
                },
              ],
              "type": "div",
            },
            null,
            null,
          ],
          "type": "div",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
